Resources/string.csv

```diff
@@ -11,735 +20,734 @@
+AntiAdminerVI,生命检测仪警告,Notify about Pulse,Оповещать о Пульсах
+AntiAdminerDL,日志警告,Notify about Journals,Оповещать о Журналах
+AntiAdminerCA,摄像头警告,Notify about Cameras,Оповещать о Камерах
+Roles,职业,Roles,Роли
+Settings,设定,Settings,Настройки
+Addons,附加效果,Add-Ons,Атрибуты
+LastResult,游戏结果,Match Results,Результат матча
+LastEndReason,结束原因,End Reason,Причина окончания
+KillLog,击杀日志,Kill Log,История убийств
+Maximum,最大人数,Max,Максимум
+RoleRate,<color=#00ff00>启用</color>,<color=#00ff00>Active</color>,<color=#00ff00>Активен</color>
+RoleOn,<color=#ffc51e>优先</color>,<color=#ffc51e>ON</color>,<color=#ffc51e>ВКЛ</color>
+RoleOff,<color=#ff6633>禁用</color>,<color=#ff6633>OFF</color>,<color=#ff6633>ВЫКЛ</color>
+Rate0,0%,0%,0%
+Rate5,5%,5%,5%
+Rate10,10%,10%,10%
+Rate20,20%,20%,20%
+Rate30,30%,30%,30%
+Rate40,40%,40%,40%
+Rate50,50%,50%,50%
+Rate60,60%,60%,60%
+Rate70,70%,70%,70%
+Rate80,80%,80%,80%
+Rate90,90%,90%,90%
+Rate100,100%,100%,100%
+Preset,预设,Preset,Сохранения
+Preset_1,预设1,Preset 1,Сохранение 1
+Preset_2,预设2,Preset 2,Сохранение 2
+Preset_3,预设3,Preset 3,Сохранение 3
+Preset_4,预设4,Preset 4,Сохранение 4
+Preset_5,预设5,Preset 5,Сохранение 5
+Standard,标准,Standard,Стандартный
+GameMode,游戏模式,Game Mode,Игровой Режим
+PressTabToNextPage,按下tab键查看更多...,Press Tab or Number for Next Page...,Нажмите Tab или Цифру чтобы перейти на следующую страницу...
+RoleSummaryText,游戏结束时玩家的职业:,Game End Player-Roles List:,Игроки и роли в конце игры:
+doOverride,特别设置%role%的任务数,Override %role%'s Tasks(At Least 1),Изменить задания у %role%
+assignCommonTasks,%role%的普通任务数,Give %role% Common Tasks,%role% имеет общие задания
+roleLongTasksNum,%role%的长任务数,%role% Long Tasks,%role% долгие задания
+roleShortTasksNum,%role%的短任务数,%role% Short Tasks",%role% короткие задания
+Format.Players,{0}人,{0},{0}
+Format.Seconds,{0}秒,{0}s,{0} с
+Format.Percent,{0}%,{0}%,{0}%
+Format.Times,{0}次,{0},{0}
+Format.Multiplier,{0}x,{0}x,{0}х
+Format.Votes,{0}票,{0},{0}
+Format.Pieces,{0}个,{0},{0}
+KillButtonText,击杀,Kill,Убить
+SniperSnipeButtonText,狙击,Snipe,Застрелить
+FireWorksExplosionButtonText,引爆,Fire,Зажечь
+FireWorksInstallAtionButtonText,设置,Set,Установить
+SerialKillerSuicideButtonText,自杀倒计时,Suicide Timer,До Суицида
+WarlockCurseButtonText,下咒,Curse,Проклинать
+AssassinKillButtonText,刺杀,Kill,Убить
+AssassinMarkButtonText,标记,Mark,Пометить
+WitchSpellButtonText,诅咒,Spell,Заклинать
+VampireBiteButtonText,吸血,Bite,Укусить
+MinerTeleButtonText,传送,Deliver,Отправиться
+ArsonistDouseButtonText,涂油,Douse,Облить
+PuppeteerOperateButtonText,操控,Manipulate,Управлять
+BountyHunterChangeButtonText,变更,Swap,До Смены
+EvilTrackerChangeButtonText,追踪,Track,Отслеживать
+DefaultShapeshiftText,变形,Shift,Превращение
+DisabledBySettings,已在设置被中禁用,Disabled by Settings,Отключено настройкой
+Disabled,禁用,Disabled,Отключено
+FailToTrack,无法追踪,Failed To Track,Не Удалось Отследить
 
 ## 作弊
-Cheat.AUM,开挂（使用AUM）
+Cheat.AUM,开挂（使用AUM）,Hang up (using AUM),Повесить трубку (используя AUM)

```

Comment:
This message will sent to host when some one cheating

Reply:
"AUM" is "Among Us Mod"?

Reply:
oh, that's "Among Us Menu", the most common cheat
https://github.com/BitCrackers/AmongUsMenu

Reply:
Wow, I didn’t even know that GitHub has a mod for AU with cheats

Reply:
im amazed same as you when i know it. lol